src/lasp_dependence_dag.erl

```diff
@@ -44,20 +53,53 @@
 %% lets us quickly delete those edges.
 -type process_map() :: dict:dict(pid(), {id(), id()}).
 
+%% Stored metadata for a lasp process.
+%% We don't store the input(s) and output vertices as that information
+%% is implicitly stored by edges in the graph.
+-record(process_metadata, {read :: function(),
```

Comment:
Does this mean that a single process only read and writes to *one* object, given that it's a single function?